src/components/MemberPanel.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { Button, SudoApplicationModal } from "../components";
+
+type Props = {
+  address: string;
+};
+
+const OwnerPanel: React.FC<Props> = (props) => {
```

Comment:
this is called OwnerPanel, but is exported in the file MemberPanel

Reply:
oops, good catch